docker-compose-net/docker-compose-net.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: guides
+title: How to Use Docker Compose with Weave Net
+---
+
+Docker Compose is used to configure multiple container apps. A Docker Compose file configures your application's services and with a single command creates and starts services. Depending on how you are using Weave Net, whether through the Docker API Proxy or by using the Weave Docker Plugin, the syntax of the Docker Compose file differs. This guide provides an overview of the different ways in which networking can be configured and also provides examples of the Docker Compose files for Weave Net. 
+
+The following topics are discussed: 
+
+  * [What You Will Use](#what-use)
+  * [Before You Begin](#before)
+  * [Attaching Containers to a Weave Network](#attaching-containers)
+  * [Docker Compose and the Weave Net Docker API Proxy](#setup-proxy)
+     * [Docker Compose File Syntax for the Weave Docker API Proxy](#compose-proxy)
+  * [Docker Compose and the Weave Net Docker Plugin](#setup-plugin)
+    * [Docker Compose File Syntax and the Weave Net Docker Plugin](#compose-plugin)
+
+##<a name="what-use"></a>What You Will Use
+
+* [Weave](http://weave.works)
+* [Weave Scope](http://weave.works/scope/index.html)
```

Comment:
"Weave Scope to see the containers and confirm that everything is working correctly" or something